include/sway/tree/view.h

```diff
@@ -268,6 +266,17 @@ void view_set_activated(struct sway_view *view, bool activated);
  */
 void view_request_activate(struct sway_view *view);
 
+/**
+ * If possible, instructs the client to change their decoration mode.
+ */
+void view_set_csd_from_server(struct sway_view *view, bool enabled);
+
+/**
+ * Updates the view's border setting when the client unexpectedly changes their
+ * decoration mode.
+ */
+void view_set_csd_from_client(struct sway_view *view, bool enabled);
```

Comment:
I tried to use "set" for the server-side and "update" for the client-side. Is it possible to rename this to `view_update_csd_from_client`?